builds/flu/flu_titers.py

```diff
@@ -189,7 +206,7 @@ def glycosylation_count(aa):
     # }
 
 
-def H3N2_scores(runner, tree, epitope_mask_file, epitope_mask_version='wolf'):
+def IAV_scores(runner, tree, mask_file, epitope_mask_version='wolf'):
```

Comment:
Feels strange to name this `IAV_scores`. Influenza A virus scores? I'm assuming the same machinery would be used for B (with masks in place). Can we rename? And why does this exist in parallel to `seasonal_flu_scores` (line 95)? This should be one method right?

Reply:
I agree that the name is strange, but less strange than `h3n2_scores` when used for h1n1 as well...